mobify/mobify-code-style
# Release v2.4.1

Status: **Ready for Review**
Owner: @marlowpayne
Reviewers: @mikenikles 

## Changes
- Cut 2.4.1 release
- Includes bug fixes inside sass-lint config file

## Todos:
- [ ] Post merge: Tag, release, and publish

### Feedback:
_none so far_

## How to Test
- Ensure base branch is correct
- Ensure changelog is updated
- Ensure version number is correctly bumped
